requirements/pip_tools.txt

```diff
@@ -8,10 +8,13 @@ click==8.0.1
     # via pip-tools
 pep517==0.10.0
     # via pip-tools
-pip-tools==6.1.0
+pip-tools==6.2.0
     # via -r requirements/pip_tools.in
 toml==0.10.2
     # via pep517
+wheel==0.36.2
```

Comment:
@Jawayria I think you might know, why is this adding `wheel`?

Reply:
@sarina We're getting them indirectly via `pip-tools` (as indicated in the requirements comment below).

https://github.com/jazzband/pip-tools/pull/889

> pip, setuptools and wheel are runtime dependencies that should
be explicitly included. If not, they may not be provided on
systems which install packages pre-built, thus these packages
should not be assumed to be installed.

XBlock `txt` requirements are not used in production, so there shouldn't be any risk, regardless.